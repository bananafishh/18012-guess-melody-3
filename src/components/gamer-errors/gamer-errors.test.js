import React from 'react';
import renderer from 'react-test-renderer';

import GamerErrors from './gamer-errors';

describe(`Компонент «GamerErrors» рендерится корректно`, () => {
  it(`Если игрок не совершил ошибок`, () => {
    const tree = renderer
      .create(
          <GamerErrors count={0}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Если игрок совершил 3 ошибки`, () => {
    const tree = renderer
      .create(
          <GamerErrors count={3}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

