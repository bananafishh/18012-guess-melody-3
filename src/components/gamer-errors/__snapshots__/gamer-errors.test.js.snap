// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Компонент «GamerErrors» рендерится корректно Если игрок не совершил ошибок 1`] = `null`;

exports[`Компонент «GamerErrors» рендерится корректно Если игрок совершил 3 ошибки 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
